### ISSUE TITLE
Accessibility: no label on form

### ISSUE THREAD

**Issue by robertmausser** [cgkineo/adapt-search#67]
### Subject of the issue/enhancement/features
"Form element need a label Element(code reference below ) 

https://www.w3.org/WAI/tutorials/forms/labels/

Location:
input
<input class=""search__textbox js-search-textbox-change"" type=""text"" placeholder="""">"

### Your environment
* version 5.8
* Chrome latest
* Windows 11

### Steps to reproduce
Check form, no label.

### Expected behaviour
Label should exist on form field

### Actual behaviour
No Label
